### PR TITLE
Use PHP 5.5 syntax

### PR DESCRIPTION
**Pull request by freekmurze**
Because it's almost 2018 😀

### CODE REVIEW

**Comment by carusogabriel** [comment]
Don't let @sisve see it, he hates CS PRs :joy:

File: views.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ For this example, let's register the view composers within a [service provider](
         {
             // Using class based composers...
             View::composer(
-                'profile', 'App\Http\ViewComposers\ProfileComposer'
+                'profile', App\Http\ViewComposers\ProfileComposer::class
```
Comment (carusogabriel) [inline views.md]: We don't need a backslash here, as it is namespaced? :thinking:

File: views.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ You may attach a view composer to multiple views at once by passing an array of
 
     View::composer(
         ['profile', 'dashboard'],
-        'App\Http\ViewComposers\MyViewComposer'
+        App\Http\ViewComposers\MyViewComposer::class
```
Comment (carusogabriel) [inline views.md]: We don't need a backslash here, as it is namespaced? :thinking:

**Comment by taylorotwell** [comment]
I avoid this because it's namespaced. Would require import or backslash.

**Pull request closed without merging**